### PR TITLE
SB-3557 Adding Support for Customized Limit Scale Down Ops/Day

### PR DESCRIPTION
**Pull request by xilongfan**
Changes are quite straightforward, limit can be applied on both table or gsi level, @wtiu-spokeo @weesan @mlispokeo, let me know if you guys see anything needs update.

### CODE REVIEW

**Comment by weesan** [comment]
Hi @xilongfan , could you please elaborate the purpose of the changes?  Also, what's the behavior when we try to change the capacity before and after your changes?  Thx.

**Comment by xilongfan** [comment]
@weesan This change is solely for let user has the ability to reserve last scaling down opportunity such that process won't use up all 4 scaling downs(if configured), nothing change related to the actual scaling behavior/conditions.

File: dynamic_dynamodb/config/__init__.py [hunk 1]
```diff
@@ -35,6 +35,8 @@
         'log_config_file': None
     },
     'table': {
+        'max_scale_down_ops_per_day': 4,
```
Comment (weesan) [inline dynamic_dynamodb/config/__init__.py]: Should max_scale_down_ops_per_day be max-scale-down-ops-per-day just to be consistent with the existing style?

File: dynamic_dynamodb/config/__init__.py [hunk 2]
```diff
@@ -99,6 +101,8 @@
         'circuit_breaker_timeout': 10000.00
     },
     'gsi': {
+        'max_scale_down_ops_per_day': 4,
```
Comment (weesan) [inline dynamic_dynamodb/config/__init__.py]: Same here.

**Pull request merged**